src/runtime_src/tools/xclbin/XclBinClass.h

```diff
@@ -55,6 +55,10 @@ class XclBin {
   void setKeyValue(const std::string & _keyValue);
   void removeKey(const std::string & _keyValue);
 
+  public:
+    static void getKeyValueComponents(const std::string & _keyValue, std::string & _domain, std::string & _key,std::string & _value);
+    static std::string findKeyAndGetValue(const std::string & _searchDomain, const std::string & _searchKey, std::vector<std::string> _keyValues);
```

Comment:
vector by value, really?  if true, then maybe it deserves a comment.?

Reply:
Good catch.  I will fix this issue.

Reply:
Fixed